tests/ltp/run_ltp_test.sh

```diff
@@ -28,7 +28,10 @@ case "$test_mode" in
 esac
 test_class="ltp"
 
-SGX_LKL_RUN_CMD=( "$SGXLKL_STARTER" $run_flag sgxlkl-miniroot-fs.img )
+SGXLKL_LTP_TSTAPP_CFG="--enclave-config=ltp_tstapp_enclave_config.json"
+SGXLKL_LTP_HOST_CFG="--host-config=ltp_host_config.json"
+
+SGX_LKL_RUN_CMD=( "$SGXLKL_STARTER" $SGXLKL_LTP_HOST_CFG $SGXLKL_LTP_TSTAPP_CFG $run_flag sgxlkl-miniroot-fs.img )
```

Comment:
Please put all env variables in double quotes similar to "$SGXLKL_STARTER"

Reply:
done